packages/zk-transfer-web/src/libs/types.ts

```diff
@@ -0,0 +1,60 @@
+export type ERCToken = Map<any, any>;
```

Comment:
This should more specifically be called an `ERC20Token` as there are many ERCs out there, standards for describing other things besides tokens. What are the keys and values? Can we be more specific about the types?

Reply:
I haven't see the structure of the ERC20Token. So that's why i give it a primitive type. I'll replace it once i see it's structure.

Reply:
Sure. It is exactly the same as the ZKToken,
they are all smart contracts that are deployed, and associated with a trading symbol like "AAA"